src/databases/OpenPMD/OpenPMDClasses/PMDFile.C

```diff
@@ -196,16 +197,48 @@ void PMDFile::ScanFileAttributes()
             this->version = buffer;
 
             delete [] buffer;
+
+            // validate openPMD standard version range
+            char version_error[1024];
+            if (VersionGreaterThan("1.0.0", this->version)) // 1.0.0>file
+            {
+                // file format too old: < 1.0.0
+                snprintf(version_error, 1024,
+                         "Standard '%s' in openPMD file is too old! "
+                         "Supported version range: >=1.0.0,<2.0.0 ",
+                         this->version.c_str());
+#ifndef TEST
+                EXCEPTION1(InvalidDBTypeException, version_error);
+#endif
```

Comment:
Do you want to enable the exception or leave dormant?

Reply:
I am not entirely sure what the convention for that one is, so I took the same approach as I found elsewhere in the database reader. You mean I should just remove the `ifndef` or use a different exception class?

Reply:
the exception will tell VisIt it should give up and try another database, so if the plugin doesn't support these cases, I think it's best to keep the exception active.

Here is a snippet from the Getting Data into VisIt manual:

```
In the event that your Initialize method cannot open the file if the file is not the right type,
or if it contains errors, or if it cannot be accessed for some other reason, the constructor
must throw an InvalidDBTypeException exception. When the
InvalidDBTypeException exception is thrown, VisIt’s database factory catches the
exception and then tries to open the file with the next matching plug-in. This procedure
continues until the file is opened by a suitable plug-in or the file cannot be opened at all
```